server/src/model/Comments.ts

```diff
@@ -1,27 +1,35 @@
 import {
   BaseEntity,
   Column,
+  CreateDateColumn,
   Entity,
   ManyToOne,
   PrimaryGeneratedColumn,
   Relation,
+  Unique,
 } from "typeorm";
 // eslint-disable-next-line import/no-cycle
 import { Item, Customer } from ".";
 
 @Entity()
+@Unique(["comments"])
```

Comment:
This will not let 2 people to have a "good product" comment

this should have both customer and item